crates/evm/src/evm/handler.rs

```diff
@@ -503,6 +509,15 @@ fn calc_diff_size<EXT, DB: Database>(
     }
 
     for (addr, account) in account_changes {
+        let db_account_size = {
+            let account = &state[addr];
```

Comment:
what's the type of state? are we sure we can access all accounts like this?

Reply:
it is hashmap, if it is journaled, which is what we need, then yes

Reply:
i wish we could use btreemap for that haha